sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinCodeVerifierTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.slang.kotlin;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class KotlinCodeVerifierTest {
+  private KotlinCodeVerifier kotlinCodeVerifier = new KotlinCodeVerifier();
+  @Test
+  public void testContainsCode() {
+    String comment = "This is a normal sentence: definitely not code";
```

Comment:
Looking at the results on SQ, rate of false positive is much better with current version than before. Some are still raised, for instance `// Loops --------------------------------------------------------------------------------------------------` will be considered as valid code. 
I also saw several times `// continue` flagged as an issue, although it is technically valid Kotlin code, it doesn't make much sense to raise issue there. WDYT about adding a check that comment contains > 1 word for it to be valid code?

Reply:
Indeed, that's a valid point. I will add this check.